.travis.yml

```diff
@@ -137,16 +126,15 @@ script:
      CPPFLAGS="" &&
      SKIP_TEST_DIRS="" &&
      if [ "$CODECOV" == "YES" ]; then
-        CPPFLAGS="--coverage";
+        CPPFLAGS="--coverage -g";
         SKIP_TEST_DIRS="miniapps/performance";
```

Comment:
Is this still necessary? Maybe we can pick up a few more percentages if we don't skip the HPC examples?

Reply:
May be worth a try to see how that looks.